slabby/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! A typed slab allocator suitable for use with `#![no_std]`.
 #![no_std]
 #![feature(unique)]
```

Comment:
I believe `feature(unique)` was _renamed_  to `feature(ptr_internals)`, so this line should be removable. Can I get you to check that this is indeed the case while I install yesterday's nightly? :)

Reply:
Yes, you're right. Removed the extra feature flag and it still builds.